src/js/calculation/draftLeaders.js

```diff
@@ -1,8 +1,6 @@
 import shuffle from 'lodash/shuffle.js';
 
 class NotEnoughLeadersError extends Error {
-    name = 'NotEnoughLeadersError';
```

Comment:
Is there a reason for removing this? This is purely a labeling thing to identify the error in a stack trace should it appear.